pytorch_lightning/loggers/mlflow.py

```diff
@@ -40,21 +19,46 @@ def any_lightning_module_function_or_hook(...):
 
 
 class MLFlowLogger(LightningLoggerBase):
-    """MLFLow logger"""
+    """
+    Log using `MLflow <https://mlflow.org>`_. Install it with pip:
+
+    .. code-block:: bash
 
+        pip install mlflow
+
+    Example:
+
+    >>> from pytorch_lightning import Trainer
```

Comment:
this shall be shifted by 4 spaces

Reply:
is it ok now?

Reply:
it is for this one but the same is in two more places
as Example is a section, the doctest shall be shifted, right?

Reply:
ok, right. check again, is it better now?